packages/@guidesmiths/cuckoojs-schematics/src/dockerfile/schema.json

```diff
@@ -44,5 +44,5 @@
       }
     }
   },
-  "required": ["directory"]
+  "required": ["directory", "buildType", "nodeVersion"]
```

Comment:
@w3dani , perhaps you don't need to include "nodeVersion" as it has a default value. What do you prefer?

Reply:
I think that's more a philosophical discussion. Is "having a default value" exclusive from "being required"? I think having both values is more meaningful, but it's ok for me to remove it from required list.

Reply:
Understood. Let's keep it as it is for now.